### PR TITLE
add support for multiple cinder backends

### PR DESCRIPTION
**Pull request by lu1as**
Add support for multiple cinder backends on the same block storage node

### CODE REVIEW

File: roles/cinder/templates/cinder.conf.j2 [hunk 1]
```diff
@@ -1,20 +1,19 @@
 # {{ansible_managed}}
```
Comment (tobikris) [inline roles/cinder/templates/cinder.conf.j2:2]:
```suggestion

{# prefix names/pools with backend type for reference in enabled_backends #}
```

File: roles/cinder/templates/cinder.conf.j2 [hunk 2]
```diff
@@ -1,20 +1,19 @@
...
 
+{% set lvm_backends = ['lvm_'] | product(cinder_lvm_pools | map(attribute='name')) | map('join') | list if 'lvm' in cinder_backends else [] %}
+{% set rbd_backends = ['rbd_'] | product(cinder_rbd_pools | map(attribute='pool')) | map('join') | list if 'rbd' in cinder_backends else [] %}
```
Comment (tobikris) [inline roles/cinder/templates/cinder.conf.j2:4]: Why do you use the pool here? Name would be more consistent.

Comment (lu1as) [reply roles/cinder/templates/cinder.conf.j2:4]: You're right, but we used the pool variable before, so I'm not sure if we run into issues with existing deployments.

Comment (tobikris) [reply roles/cinder/templates/cinder.conf.j2:5]: That makes sense, we can keep it.

**Pull request merged**